docs/CLI_Reference/oras_repo_ls.md

```diff
@@ -0,0 +1,38 @@
+## oras repo ls
+
+List the repositories under the registry.
+
+```bash
+oras repo ls [flags] <registry>
+```
+
+## Examples
+
+Example - List the repositories under the registry:
+
+```bash
+  oras repo ls localhost:5000
+```
+
```

Comment:
There is a new example:
~~~suggestion
Example - List the repositories under a namespace in the registry:
```bash
  oras repo ls localhost:5000/example-namespace
```
~~~

Reply:
Added this example